### PR TITLE
chore(docs): add markToTrack docs and fix isDeepChanged docs

### PR DESCRIPTION
**Pull request by YPAzevedo**
create docs for `markToTrack` and small fix to `isDeepChanged` docs

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -303,6 +303,34 @@ export const getUntrackedObject = <T>(obj: T): T | null => {
...
+ * const orginal = { a: "1", c: "2", d: nested };
+ * const affected = new WeakMap();
+ *
+ * const proxy = createDeepProxy(orginal, affected);
+ *
+ * proxy.d.e
+ *
+ * isDeepChanged(original, { d: { e: "3" } }, affected) // true
+ */
 // mark object to track or not (even if it is not plain)
```
Comment (dai-shi) [inline src/index.ts:334]:
```suggestion
```

Let's remove this.

File: src/index.ts [hunk 2]
```diff
@@ -303,6 +303,34 @@ export const getUntrackedObject = <T>(obj: T): T | null => {
...
+/**
+ * Mark object to be tracked.
+ *
+ * This function marks an object that will be passed into `createDeepProxy`
+ * as marked to track or not. By default only Array and Object are marked to track,
+ * so this is useful for example to mark a class instance to track or to mark a object
+ * to be untracked when creating your proxy.
+ *
+ * @param {object} obj - Object to mark as tracked or not.
+ * @param {mark} boolean - Boolean indicating wether you want to track this object or not..
```
Comment (dai-shi) [inline src/index.ts:315]:
```suggestion
 * @param {mark} boolean - Boolean indicating whether you want to track this object or not.
```

**Review by dai-shi — APPROVED** [review]
looks good.

**Comment by YPAzevedo** [comment]
Thanks for the updates @dai-shi i totally missed the typos 😅

**Comment by YPAzevedo** [comment]
Let me know if it needs any more changes 👍

**Comment by YPAzevedo** [comment]
@dai-shi added one more fix to the isDeepChanged in this PR

**Pull request merged**